api/machine/machine.proto

```diff
@@ -776,3 +778,18 @@ message GenerateConfigurationResponse {
   repeated bytes data = 2;
   bytes talosconfig = 3;
 }
+
+// NetworkInterface describes a single network interface information.
+message NetworkInterface {
+  repeated string addrs = 1;
+  string name = 2;
+  string mac = 3;
+  int32 mtu = 4;
+  bool up = 5;
+}
```

Comment:
We already have an interface definition in `network.proto`. IMHO we should either a) import that here and add any fields we need to it or 2) have the maintenance service implement the network server (just embed the default unimplemented server) and the implement the interfaces RPC. Given that the pattern we have used so far is to embed the unimplemented server, I would lean towards that.

Reply:
oh... It's completely the same there. Yeah that makes perfect sense.